products/windows.md

```diff
@@ -9,236 +9,259 @@ releasePolicyLink: https://learn.microsoft.com/lifecycle/products/?terms=Windows
     link: https://learn.microsoft.com/lifecycle/announcements/windows-10-1803-1809-end-of-servicing
 
--   releaseCycle: "10, version 1803 (W)"
+-   releaseCycle: "10-1803-W"
+    releaseLabel: "10 1803 (W)"
     releaseDate: 2018-04-30
     support: 2019-11-12
     eol: 2020-05-11
     latest: 10.0.17134
     link: https://learn.microsoft.com/lifecycle/announcements/windows-10-1803-end-of-servicing
 
--   releaseCycle: "10, version 1709 (E)"
+-   releaseCycle: "10-1709-E"
+    releaseLabel: "10 1709 (E)"
     releaseDate: 2017-10-17
     support: 2020-10-13
     eol: 2020-09-13
     latest: 10.0.16299
     link: https://learn.microsoft.com/lifecycle/announcements/windows-10-1709-end-of-servicing
 
--   releaseCycle: "10, version 1709 (W)"
+-   releaseCycle: "10-1709-W"
+    releaseLabel: "10 1709 (W)"
     releaseDate: 2017-10-17
     support: 2019-04-09
     eol: 2019-04-09
     latest: 10.0.16299
     link: https://learn.microsoft.com/lifecycle/announcements/windows-10-1709-end-of-servicing
 
--   releaseCycle: "10, version 1703 (E)"
+-   releaseCycle: "10-1703-E"
+    releaseLabel: "10 1703 (E)"
     releaseDate: 2017-04-11
     support: 2019-10-08
     eol: 2019-10-08
     latest: 10.0.15063
     link: https://techcommunity.microsoft.com/t5/windows-it-pro-blog/end-of-service-reminders-for-windows-10-versions-1703-and-1803/ba-p/903715
 
--   releaseCycle: "10, version 1703 (W)"
+-   releaseCycle: "10-1703-W"
+    releaseLabel: "10 1703 (W)"
     releaseDate: 2017-04-11
     support: 2018-10-09
     eol: 2018-10-09
     latest: 10.0.15063
     link: https://techcommunity.microsoft.com/t5/windows-it-pro-blog/end-of-service-reminders-for-windows-10-versions-1703-and-1803/ba-p/903715
 
--   releaseCycle: "10, version 1607"
+-   releaseCycle: "10-1607"
+    releaseLabel: "10 1607"
     releaseDate: 2016-08-02
     support: 2021-10-12
     eol: 2026-10-13
     lts: true
     latest: 10.0.14393
     link: https://learn.microsoft.com/windows/release-health/supported-versions-windows-client#enterprise-and-iot-enterprise-ltsbltsc-editions
 
--   releaseCycle: "10, version 1607 (E)"
+-   releaseCycle: "10-1607-E"
+    releaseLabel: "10 1607 (E)"
     releaseDate: 2016-08-02
     support: 2019-04-09
     eol: 2019-04-09
     latest: 10.0.14393
     link: https://learn.microsoft.com/lifecycle/announcements/windows-10-1607-end-of-servicing
 
--   releaseCycle: "10, version 1607 (W)"
+-   releaseCycle: "10-1607-W"
+    releaseLabel: "10 1607 (W)"
     releaseDate: 2016-08-02
     support: 2018-04-10
     eol: 2018-04-10
     latest: 10.0.14393
     link: null
 
--   releaseCycle: "10, version 1511 (E)(W)"
+-   releaseCycle: "10-1511"
+    releaseLabel: "10 1511"
     releaseDate: 2015-11-10
     support: 2017-10-10
     eol: 2017-10-10
     latest: 10.0.10586
     link: https://learn.microsoft.com/lifecycle/announcements/windows-10-1511-end-of-servicing
 
--   releaseCycle: "10, version 1507"
+-   releaseCycle: "10-1507"
+    releaseLabel: "10 1507"
```

Comment:
10 1507 is now duplicated.

Reply:
Solved with 7a09b7d1df4efe13102d3ed2ad15420ea39799fd